hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -216,6 +216,10 @@ public final class HddsConfigKeys {
   // format hh:mm:ss, representing hour, minute, and second
   public static final String HDDS_X509_CA_ROTATION_TIME_OF_DAY_DEFAULT =
       "02:00:00";
+  public static final String HDDS_X509_ROOTCA_CLIENT_POLLING_FREQUENCY =
+      "hdds.x509.rootca.client.polling.frequency";
```

Comment:
I would suggest user "interval" instead of "frequency" here, to be consistent with existing other properties.

Reply:
@Galsza I believe this comment is for your patch at #4961 ;)

Reply:
Yes, that's a better word, I'm going to fix it